client/src/components/uploads/uploadProfile/index.tsx

```diff
@@ -55,6 +56,8 @@ const UploadProfile = ({ isOpen, resource, onClose, onUpload }: Props) => {
   );
   const [uploadingProfile, setUploadingProfile] = React.useState(false);
 
+  const validate = validator(fhirSchema);
```

Comment:
this will re-compile the json schema each time the component is re-rendered, I would prefer having it outside of the component.